app/server/lib/OIDCConfig.ts

```diff
@@ -147,9 +156,14 @@ export class OIDCConfig {
 
   public async getLogoutRedirectUrl(req: express.Request, redirectUrl: URL): Promise<string> {
     // For IdPs that don't have end_session_endpoint, we just redirect to the logout page.
-    if (this._client.issuer.metadata.end_session_endpoint === undefined) {
+    if (this._skipEndSessionEndpoint) {
       return redirectUrl.href;
     }
+    if (this._client.issuer.metadata.end_session_endpoint === undefined) {
```

Comment:
Would it be possible to move this check to initialization instead? That way the person setting up will know about the issue right away, rather than only when they remember to test logouts.

Reply:
:100: Done